### ISSUE TITLE
styles needed in workspace to pass color contrast on Page layout

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#725]
The [Page layout](https://pf-next.com/layouts/Page/examples/) currently does not pass color contrast check due to styles currently defined. 

~~Since this failure is only in the context of the example page, and not an issue with the patternfly css itself, then additional styles should be added to the workspace to alter the background color of the example.~~ 
To fix this issue, the text that's failing color contrast can be removed from the example.

To learn more about how to build and test locally, check out: https://github.com/patternfly/patternfly-next#testing-for-accessibility 

@mceledonia is the visual design contact that would be able to help with answers regarding colors and color contrast.

**Comment by jgiardino** [comment]
The text that was failing was removed in #1057
